.github/workflows/markdown-altimetry.yml

```diff
@@ -0,0 +1,51 @@
+name: notebook_to_md (altimetry)
+
+on:
+  # External webhook trigger
+  repository_dispatch:
+   types: [push-notebook]
```

Comment:
We probably want to change the trigger for this. This webhook approach was used to trigger the notebook action on the coast-site repo once a merge to develop was done on the coast repo,

Reply:
@roje-bodc  changed to on pull request